x-pack/plugins/ml/public/timeseriesexplorer/components/annotation_description_list/_index.scss

```diff
@@ -0,0 +1,17 @@
+// SASSTODO: This is based on the overwrites used in the Filters flyout to match the existing style.
+
+// SASSTODO: Dangerous EUI overwrites
+.euiDescriptionList.euiDescriptionList--column.ml-annotation-description-list {
+  .euiDescriptionList__title {
```

Comment:
I'll make a ticket in EUI for this. We should add some props here so you can manage it if you are using the object array version of description list.

Reply:
cool, thanks!